index.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="description" content="the author Portfolio" />
+    <meta name="author" content="Arthur Emanuel Guerra Iturres" />
+    <link
+      rel="icon"
+      href="src/assets/media/tab-app-icons/illustration-tab-icon.png"
+    />
+    <link
+      rel="stylesheet"
+      href="src/assets/styles/main-stylesheets/index.css"
+    />
+    <title>Arthur.Iturres | Portfolio</title>
+  </head>
+  <body id="go-top">
+    <header>
+      <nav id="my-navbar" class="my-navbar">
+        <ul class="hidden">
+          <li>
+            <a class="my-navbar__link" href="#go-top">Portfolio</a>
+          </li>
+          <li><a class="my-navbar__link" href="#about-section">About</a></li>
+          <li>
+            <a class="my-navbar__link" href="#contact-section">Contact</a>
+          </li>
+        </ul>
+        <span class="close-my-navbar__btn my-btn hidden" data-open-my-navbar>
+          <i class="fa-solid fa-x"></i>
+        </span>
+        <div class="open-my-navbar">
+          <span class="open-my-navbar__btn my-btn" data-close-my-navbar>
+            <i class="fa-solid fa-bars"></i>
+          </span>
+        </div>
+      </nav>
+    </header>
+    <section class="hero-section">
+      <div class="hero-section__title--wrapper">
+        <h1 class="hero-section__title">Hey there.</h1>
+        <h2 class="hero-section__title">I'm Arthur.</h2>
+      </div>
+      <h3 class="hero-section__subtitle">I'm a software developer</h3>
+      <p class="hero-section__intro">
+        I can help you build a product, feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don't hesitate to contact me.
+      </p>
```

Comment:
You're doing an amazing job with your project so far! I have a suggestion that could help make it even better if you're open to it:

- [ ] **[OPTIONAL]** To align with the Figma Template, please consider using a `font-weight` of 400 for this paragraph instead of 500. 🎯

### Figma design:
![image](https://user-images.githubusercontent.com/107952724/235011737-5a6f45e7-12e3-4cda-8794-5de5db3aabac.png)


### Your design:
![image](https://user-images.githubusercontent.com/107952724/235012023-f2869973-8571-45f1-aacd-e4ebbcc62d4e.png)

Reply:
Hi @taldr27 ! Thank you so much for your reply and advice! 😊I will set the font weight to 400✅ right away. It does look accurate with the recommended value.